src/mattack_actors.h

```diff
@@ -98,25 +98,26 @@ class gun_actor : public mattack_actor
         /*@{*/
         /** Turrets may need to expend moves targeting before firing on certain targets */
 
-        int targeting_cost = 100; /** moves consumed before first attack can be made */
+        int targeting_cost = 100; /** Moves consumed before first attack can be made */
 
-        bool require_targeting_player = true; /** by default always give player some warning */
```

Comment:
These capitalizations double the line count and break git blame for no tangible benefit. The class is clearly commented so there isn't any need to do this.